package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodebb-plugin-ns-points",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
Please, do not include version change as a part of PR

Reply:
Didn't know before.. fixed